README.md

```diff
@@ -2,30 +2,86 @@ Core ML Community Tools
 =======================
 
 Core ML community tools contains all supporting tools for Core ML model
-conversion and validation. This includes scikit-learn, LIBSVM, Caffe,
-Keras and XGBoost.
+conversion, editing and validation. This includes scikit-learn, LIBSVM, Caffe,
+Keras, Tensorflow and XGBoost.
 
-coremltools 3.0
----------------
-[Release notes](https://github.com/apple/coremltools/releases/)
+# Table of contents
 
+* [Release information](#Release-Information)
+* [Neural network conversion guide](#Neural-network-conversion-guide)
+* [Core ML specification](#Core-ML-specification) 
+* [coremltools documentation](#coremltools-documentation-and-examples)
+* [Installation](#Installation)
 
-👍👎 Please take this quick poll and let us know how you liked this release, [here](https://github.com/apple/coremltools/blob/master/release-feedback.md)!
+
+
+## Release information
+
+Latest version of coremltools is 3.1  
```

Comment:
You should ideally not have this in the readme because it will quickly get stale. I think just pip install coremltools should be enough. Also, why do we have a -U

Reply:
`-U` gets the latest non-beta version automatically.

Reply:
If we decide only keep one, I'd vote for `pip install -U coremltools` (or `pip install --upgrade coremltools`). Since many situations, the issue reported might get fixed in the latest version where the readme might behind, and the upgrade is the first thing we suggest people try.

Reply:
Removed the release info section